schemas/EGA.experiment.json

```diff
@@ -2,12 +2,324 @@
     "properties": {
+      "object_id": {
+        "type": "object",
+        "title": "Object's IDs block",
+        "description": "Node containing the main identifiers of the object (e.g. alias, center_name...), inherited from the common definitions.", 
+        "allOf": [
+          {
+            "title": "Inherited object_core_id object",
+            "$ref": "https://github.com/EbiEga/ega-metadata-schema/tree/main/schemas/EGA.common-definitions.json#/definitions/object_core_id"
+          },
+          {
+            "title": "Check that experiment EGA ID (EGAX) is correct",
+            "properties": {
+              "ega_accession": {
+                "$ref": "https://github.com/EbiEga/ega-metadata-schema/tree/main/schemas/EGA.common-definitions.json#/definitions/EGA-experiment-id-pattern"
+              }
+            }
+          }
+        ]
+      },  
+      
+      "schema_descriptor": {
+        "title": "Schema descriptor node",
+        "description": "Inherited schema descriptor node containing metadata about the schemas and standards used to create the JSON document itself.",
+        "$ref": "https://github.com/EbiEga/ega-metadata-schema/tree/main/schemas/EGA.common-definitions.json#/definitions/schema_descriptor"
+      },
+
+      "object_title": {
+        "type": "string",
+        "title": "Title of the experiment",
+        "description": "An informative experiment title that should serve as an overview of the experiment, including: used technology, samples, purpose... (e.g. 'Affymetrix-X microarray of human breast cancer cell line MCF-7 treated with tamoxifen compared with untreated controls'). This short text can be used to call out experiment records in searches or in displays.",
+        "examples": [ "Affymetrix-X microarray of human breast cancer cell line MCF-7 treated with tamoxifen compared with untreated controls" ]
+      },
+
+      "object_description": {
+        "type": "string",
+        "title": "Description of the experiment",
+        "description": "An in-depth description of the biological relevance and intent of the experiment, including the experimental workflow.",
+        "examples": [ "The experiment was conducted with the objective of... ...and for that purpose we compared untreated controls against..." ]
+      },
+
+      "sample_number": {
+        "type": "integer",
+        "title": "Number of samples of the experiment",
+        "description": "Number of samples included in the Assay (i.e. pooled into one single microarray, labelled differently). One sample corresponds to one biological replicate [EFO:0002091] (e.g. genetic content from a single cell, a tissue, buccal swab, etc.) from a single individual or from several individuals. Shall not be mistaken for technical replicates [EFO:0002090] being used several times (see https://www.ebi.ac.uk/seqdb/confluence/pages/viewpage.action?spaceKey=EGA&title=Sample+Representation).",
+        "examples": [ 30 ]
+      },
+
```

Comment:
I suggest removing this attribute. We should instead calculate sample number from the submitted metadata. By also having this attribute, we risk having inconsistency within the metadata.

Reply:
Done: https://www.ebi.ac.uk/panda/jira/browse/EE-2344